ocs_ci/ocs/monitoring.py

```diff
@@ -0,0 +1,82 @@
+import logging
+
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.utility import templating
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pvc import get_all_pvcs
+from ocs_ci.ocs.resources.pod import get_pod_obj
+from tests import helpers
+
+logger = logging.getLogger(__name__)
+
+
+def create_configmap_cluster_monitoring_pod(sc_name):
+    """
+    Create a configmap named cluster-monitoring-config
+    and configure pvc on monitoring pod
+
+    Args:
+        sc_name (str): Name of the storage class
+    """
+    logger.info("Creating configmap cluster-monitoring-config")
+    config_map = templating.load_yaml_to_dict(
+        constants.CONFIGURE_PVC_ON_MONITORING_POD
+    )
+    config_map['data']['config.yaml'] = config_map['data']['config.yaml'].replace(
```

Comment:
in case csi-rbd will change in the future this code will fail.
Just add the change you want without replacing

Reply:
```
{'apiVersion': 'v1', 'kind': 'ConfigMap', 'metadata': {'name': 'cluster-monitoring-config', 'namespace': 'openshift-monitoring'}, 'data': {'config.yaml': 'prometheusK8s:\n  volumeClaimTemplate:\n    metadata:\n      name: my-prometheus-claim\n    spec:\n      storageClassName: csi-rbd\n      resources:\n        requests:\n          storage: 40Gi\nalertmanagerMain:\n  volumeClaimTemplate:\n    metadata:\n      name: my-alertmanager-claim\n    spec:\n      storageClassName: csi-rbd\n      resources:\n        requests:\n          storage: 40Gi\n'}}
```
From config.yaml it is string. If we convert them to dict and change the key:value as we need and send a request to create configMap it fails(pod doesn't respin and pvc won't be created). So used to string replace here, any suggestions how to handle this??

Reply:
you can yaml.load(config_map['data']['config.yaml']) and than change the value you want in the ordinary way.
But up to you - if you want to leave it that way is also fine

Reply:
Thanks Raz, It works here and changed accordingly